packages/dds/merge-tree/src/test/testClient.ts

```diff
@@ -431,6 +431,22 @@ export const createRevertDriver =
     };
 };
 
+export function debugDumpTree(tree: MergeTree) {
```

Comment:
i'd leave this defined on TestClient. we don't care about bundle size of testClient.ts since it's test-only code, and making it a free function on the module will make it annoying to call while debugging without editing the code and recompiling (which is the main place where this is useful).